### DISCUSSION TITLE
Complete the fields of a relationship field during editing a row

### DISCUSSION THREAD

**Discussion by pigeonflight** [Budibase/budibase#3956] [Help] [answered]
I'm wondering if there is a way to do this in budibase.
When editing the row of a table I'm looking for an interface that allows me to add and complete the fields of a related row in a different table.

## example use case:
Football teams have multiple managers. I should be able to add a manager while editing the football team.
I have a `manager` table which includes details about the manager (name, starting year and ending year, date of birth, country of birth)
I have a `team` table which includes all the teams and a relationship field that allows many managers to be associated with a single team.

- It should be possible to add a new team, e.g Arsenal
- while adding or editing that team I want an option to add a manager and fill out their details (Arsène Wenger, 1998 - 2018, October 22, 1949, France).
- it should return me to editing the team (Arsenal) but now I see Wenger listed as one of the managers.

**Comment by mjashanks** [comment, marked as answer]
Hi David. This should describe the basics of what you need to do: https://docs.budibase.com/guides/create-a-child-record-from-the-parent

Let me know if there's anything that it misses :)

**Reply by hotemetoot** [reply]
Hi, when I click on the link I get a 404 error. Has this guide been moved somewhere?

**Comment by hendrikjap** [comment]
Hi Michael @mjashanks, what if we want to create customer and notes at the same time (in the same screen). For example, to create quotation you need two tables in db but in the screen they are together 
Quotation (quoteNo, Date, and Customer) and QuotationItem (item/service, qty, and price). I tried to modify your sample by using filtering to parent but it's not working.

**Reply by mjashanks** [reply]
At this time, you cannot create the Parent and Child records at the same time, as the parent needs to exist for the relationship to be made.

However, once we have [this issue](https://github.com/Budibase/budibase/issues/3344) sorted (end of this month), you will have a workaround.

The workaround will be
1. Setup your "New Customer" screen, with all the customer's form fields.
2. Add a text field to the form, called "Notes"
3. In the Buttons `Define Actions`, Instead of `Save Row`, use `Run Automation`
4. Pass all of the customer fields into the automation, including your `Notes` Field
5. If your Automation, create the customer record in the first step, then create the Notes record in the second step (using the customer `_id` from the first step)

**Reply by hendrikjap** [reply]
Look forward to it!
